deployment-templates/compute-engine/server/forseti-instance-server.py

```diff
@@ -23,9 +23,27 @@ def GenerateConfig(context):
         "git clone {src_path}.git".format(
             src_path=context.properties['src-path']))
 
-    FORSETI_VERSION = (
-        "git checkout {forseti_version}".format(
-            forseti_version=context.properties['forseti-version']))
+    if context.properties['matching_patches_query']:
+        CHECKOUT_FORSETI_VERSION = (
+        """matches=$(git tag -l {matching_patches_query})
+matches=(${{matches//;/ }})
```

Comment:
This bash script is a very long string, and would be worth centralizing in one place, so that it's easier to change in the future.

How about creating a constants.py in this top-level directory?
https://github.com/GoogleCloudPlatform/forseti-security/tree/dev/deployment-templates/compute-engine

Reply:
Okay to just add comments to update in both files.